### PR TITLE
removing airflow jobs 

### PR DESCRIPTION
**Pull request by wcbeard**
updating version number

### CODE REVIEW

**Review by acmiyaguchi — APPROVED** [review]
Can you summarize briefly what was added and removed in this PR for future reference? Otherwise, this looks good to me -- the job will presumably be added to mozetl for scheduling.

**Comment by wcbeard** [comment]
The main net change was removing the airflow job and tests, since I'll be putting them into the mozetl repo. There are additional minor fixes and formatting changes, and I incremented the version number to v0.1.3 as I intend to push this to pypi.
